shell/platform/android/android_shell_holder.cc

```diff
@@ -140,6 +141,12 @@ AndroidShellHolder::~AndroidShellHolder() {
   FML_CHECK(pthread_key_delete(thread_destruct_key_) == 0);
 }
 
+WindowData AndroidShellHolder::getDefaultWindowData() {
```

Comment:
Given that this doesn't rely on object state at all, this could be a static function local to the TU.

Reply:
(Either way this function should start with a capital though)

Reply:
On third thought, if you don't expect this to change significantly in the future or be called from a ton of places, I'd probably just inline the variable declaration and field initialisation at the usage site above.

Reply:
I separated this out to a method for readability. We might want to fill in more default value for add 2 app, for example, viewport metrics.